tests/counter/counter.test.ts

```diff
@@ -0,0 +1,78 @@
+import { initSimnet, tx } from "@hirosystems/clarinet-sdk";
+import { Cl } from "@stacks/transactions";
+import fc from "fast-check";
+import { describe, expect, it } from "vitest";
+
+import { counterCommands } from "./CounterCommands";
+
+const accounts = simnet.getAccounts();
```

Comment:
So that's all possible using `vitest-environment-clarinet`?

Reply:
Yep, the `vitest-environment-clarinet` does make the `simnet` available globally.
It's not mandatory to use it but it does take care of some stuff (such as collection cost and coverage reports)